Remove prefixes for grab and grabbing cursor values.

nchevobbe opened the pull request:
Those prefixes are not needed in the latest version of Chrome
and Safari, so we can safely remove them.

Fixes #2618.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/uber/deck.gl?pullRequest=2629) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/uber/deck.gl?pullRequest=2629) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/uber/deck.gl?pullRequest=2629) it.</sub>

nchevobbe commented:
Everything looks cool. Could someone review these changes?

Pessimistress commented on modules/core/src/lib/deck.js:
```diff
@@ -31,17 +31,14 @@ import {Stats} from 'probe.gl';
 import {EventManager} from 'mjolnir.js';
 
 import assert from '../utils/assert';
-import VENDOR_PREFIX from '../utils/css-vendor-prefix';
```
I believe this file can be safely removed.

Pessimistress commented on modules/react/src/utils/cursors.js:
```diff
@@ -18,15 +18,8 @@
 
-// import browser from 'bowser';
-
-const PREFIX = '-webkit-';
```
I don't see this file being imported anywhere. Just remove?

nchevobbe commented:
Thanks for the reviews. I deleted all unused file, inlined `grab` and `grabbing` in the only place it was used, and removed the `CURSOR` object which has no use (`POINTER` was not used anywhere).
